types/scroll-to-element/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for scroll-to-element 2.0
+// Project: https://github.com/willhoag/scroll-to-element
+// Definitions by: Kirill Kvashonin <https://github.com/kirillurgant>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Options {
+    offset?: number;
```

Comment:
This doesn't say it defaults to something so should not be optional property. https://github.com/willhoag/scroll-to-element#offset--number